network/geoblock.sh

```diff
@@ -32,10 +32,6 @@ else
     # Removal
     find /var/scripts -type f -regex \
 "$SCRIPTS/202[0-9]-[01][0-9]-Maxmind-Country-IPv[46]\.dat" -delete
-    if is_this_installed jq
-    then
-        apt purge jq -y
```

Comment:
Why not, `install_if_not()`?

Reply:
Maybe in a follow up but I actually removed those lines because jq is no longer part of the script.